### PR TITLE
Noh doc postprocessing sw

### PR DESCRIPTION
**Pull request by nilsolav**
Added a chapter on the use of the convention for typical post processing steps.

### CODE REVIEW

**Review by gavinmacaulay — CHANGES_REQUESTED** [review]
This commit changes every line in the document. Perhaps the line endings changed? Please adjust the commit so that we can see what lines actually changed.

File: SONAR-netCDF4/docs/crr341.adoc [hunk 1]
```diff
@@ -96,6 +96,16 @@ This document and the convention that it describes will change over time to impl
 
 The convention version number will always be included in the title of the document. The document version number will always be found in <<_revision_history>>.
 
+== The recommended use of the SONAR-netCDF4 convention in post processing software
+
+There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the standard is intended to be used to facilitate the interchangeability. Note that the steps may be different between software solutions, and the convention only recommend how to store the data for the steps that are most common and we do not address how and which order to process your data.
```
Comment (gavinmacaulay) [inline SONAR-netCDF4/docs/crr341.adoc]:
```suggestion
There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the standard is intended to be used to facilitate this interchangeability. Note that the steps may be different between tools, and that the convention only recommends how to store the data for the steps that are most common. We do not address how and in which order to process your data.
```

File: SONAR-netCDF4/docs/crr341.adoc [hunk 2]
```diff
@@ -96,6 +96,16 @@ This document and the convention that it describes will change over time to impl
...
+
+Exchange of information between different parts in the processing chain is becoming increasingly important. This is particularly relevant for emerging deep learning and machine learning frameworks, that has hardware requirements that often are different than the computers that run the traditional desktop applications. Processing tools to be run on autonomous platforms are also a need, which often require a different suite of software solutions, typically tailored to low power situations.
```
Comment (gavinmacaulay) [inline SONAR-netCDF4/docs/crr341.adoc]:
```suggestion
Exchange of information between different parts in the processing chain is becoming increasingly important. This is particularly relevant for emerging deep learning and machine learning frameworks, that have hardware requirements that are often different to the computers that are used for traditional desktop applications. Processing tools that run on autonomous platforms are also a need, which often require a different suite of software solutions, typically tailored to low power situations. 
```

File: SONAR-netCDF4/docs/crr341.adoc [hunk 3]
```diff
@@ -110,6 +120,10 @@ Each SONAR-netCDF4 file is intended to store data from one sonar mounted on one
 
 A design principle of SONAR-netCDF4 has been to focus on describing the acoustic backscatter data, not the overall purpose and context of why the data were collected. Such broader metadata are better stored in separate metadata systems and schema (e.g. citenp:[iso2014,ices2016]).
 
+Figure 1. Typical steps in a postprocessing chaing for acoustical water column data. (TODO: Link these boxes to the groups in the convention and check if there are other needs)
```
Comment (gavinmacaulay) [inline SONAR-netCDF4/docs/crr341.adoc]:
```suggestion
.Typical steps in a postprocessing chain for acoustical water column data. (TODO: Link these boxes to the groups in the convention and check if there are other needs)
[[postprocessing_data_flow_figure]]
image::postprocessingDataFlowFigure.svg[align="center"]

```

File: SONAR-netCDF4/docs/crr341.adoc [hunk 4]
```diff
@@ -96,6 +96,16 @@ This document and the convention that it describes will change over time to impl
...
+
+Exchange of information between different parts in the processing chain is becoming increasingly important. This is particularly relevant for emerging deep learning and machine learning frameworks, that has hardware requirements that often are different than the computers that run the traditional desktop applications. Processing tools to be run on autonomous platforms are also a need, which often require a different suite of software solutions, typically tailored to low power situations. 
+
+Typical steps in the postprocessing chain of marine sonar data consist of preprocessing and noise removal, bottom detection, classification, and integration (Figure 1). Here we outline the different groups in the convention that can support the input and output from these steps. If a processing software step can read the data from the standard, process the information and write back to the data using the convention, other software can pick it up and process it further.
```
Comment (gavinmacaulay) [inline SONAR-netCDF4/docs/crr341.adoc]:
```suggestion
Typical steps in the postprocessing chain of marine sonar data consist of preprocessing and noise removal, bottom detection, classification, and integration (<<postprocessing_data_flow_figure>>). Here we outline the different groups in the convention that can support the input and output from these steps. If a processing software step can read the data, process the information and write back to the data using the convention, other software can pick it up and process it further. 
```

File: SONAR-netCDF4/docs/crr341.adoc (hunk 1 above)
Comment (gavinmacaulay) [inline SONAR-netCDF4/docs/crr341.adoc]:
```suggestion
There are several processing steps that are common among the different postprocessing tools used by the community, and there is a growing need to use the different tools interchangeably. This chapter describes a few standard processing steps and outlines how the convention is intended to be used to facilitate the interchangeability. Note that the steps may be different between tools, and the convention only recommends how to store the data for the steps that are most common. We do not address how and in which order to process your data.
```

**Review by gavinmacaulay — APPROVED** [review]

**Comment by gavinmacaulay** [comment]
Looks fine. Some polishing probably needed, but this can happen in prep for a new release of the convention.

**Comment by nilsolav** [comment]
Yes, let's do the language editing later. What is missing is the link to the groups later in the document.

**Pull request merged**